src/log.js

```diff
@@ -583,7 +583,11 @@ export function user(opt_element) {
  */
 function getUserLogger(suffix) {
   if (!logConstructor) {
-    throw new Error('failed to call initLogConstructor');
+    if (getMode().test || getMode().localDev) {
```

Comment:
can we add a comment explaining why we need it for tests (randomization of modules)